### ISSUE TITLE
Create a sample demonstrating how to render local timestamps in AMP

### ISSUE THREAD

**Issue by sebastianbenz** [ampproject/amp-by-example#719]
Demo using amp-list and variable substitutions:

```
  <amp-list width="100" height="24" src="/time.json?timezone=TIMEZONE">
    <template type="amp-mustache" id="amp-template-id">
      {{formattedLocalTime}}
    </template>
  </amp-list>
```

Also demo how to handle a large number of timestamps using fetch batching. //cc @ericlindley-g

**Comment by ericlindley-g** [comment]
Good call — maybe the docs should have a callout recommending batch fetching wherever there is more than one piece of data fetched for amp-list.  (definitely when fetching more than one timestamp, but also when fetching, say, both a timestamp and a list of recommended articles — assuming they are fetched from the same backend/url)

/cc @cramforce for visibility on this sample as an example very-fresh data

**Comment by kul3r4** [comment]
@aghassemi this may be related to [amp-util-date](https://github.com/ampproject/amphtml/issues/10710)

**Comment by aghassemi** [comment]
yes, the new component will be a client-side solution for this. server-side solution still fine for cases where the client-side component can't handle (i.e. very custom formatting like "Good Afternoon!")

**Comment by QES** [comment]
@sebastianbenz did anything happen to this, I was looking at it as a way to change dates on a list of match day kickoff times from UTC to client local times this can be up to 200 times (but only 10 different times) however working out how to batch those times in a coherent way is an interesting problem.

Did you have any thoughts on how you set the time to be altered if it comes (as in my case from a lost of start times) or in the use case that @ericlindley-g mentions a list of articles and you want to chance the published time to be the local user times.

#10837 takes a slightly different approach but for multiple times I wonder if an addition to time-ago might be less complex.

**Comment by sebastianbenz** [comment]
@QES wouldn't amp-timeago solve your problem? If not, what's missing?

**Comment by QES** [comment]
Hi time-ago displays a fuzzy time - 8 minutes ago or the time as input I don’t think it allows for the time to be shown in other formats.

I have played with timeago to learn how it works on a demo page https://amp.sportsmole.co.uk/sw3/ but what I need to do is take simple times “3pm” and show them in the users local time zone. 

Time-ago might easily be modified to do this with:

A) an option to format the time if not fuzzy (cutoff) from memory you have to format the time in a standard way to get the fuzzy response.  So when the time is fully displayed a way to format it as 3pm or 3:15pm or 15:00 or 12:30 or whatever the designers decree.

B) A way to tell time-ago to use the local timezone details and apply that to the displayed time.

An example of the page I am working with might be https://amp.sportsmole.co.uk/live-scores/2017-11-18.html next week where if you are viewing from Europe the local time for you is all one hour later etc as a page it would just need a explanation of the times shown.

It’s one of those things that would be nice to do if the overhead of doing so doesn’t make the page too heavy as what is currently 4 characters (up to 200 times) could become quite a lot of html with time-ago and even more with the other example being suggested.

It’s almost more interesting to look at it as a substitution solution but that isn’t ideal either as there is a delay while loading an amp-list element.
